starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +22,59 @@ var words = [
   'crackpot'
 ];
 
+function findLongestWord(wordsArray){
+  let longWord = "";
+  for (let i = 0; i < wordsArray.length; i++){
+    if (wordsArray[i].length > longWord.length){
+      longWord = wordsArray[i];
+    }
+  }
+  return longWord;
+}
+
+let longestWord = findLongestWord(words);
+console.log(longestWord);
+
+
 // Calculating a Sum
 
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+//Using .reduce()
+function sumArray(total, num){
+  return total + num;
+}
+
+console.log(numbers.reduce(sumArray));
+
+//Using FOR Loop
+
+function sumArrayFor(numArray){
+  let sum = 0;
+  for (let i = 0; i < numArray.length; i++){
+    sum += numArray[i];
+  }
+  return sum;
+}
+
+let sumUp = sumArrayFor(numbers);
+console.log(sumUp);
+
 // Calculate the Average
 
 var numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
+function averageNumbers(numArray){
+  let sum = 0;
+  for (let i = 0; i < numArray.length; i++){
+    sum += numArray[i];
+  }
+  return sum/numArray.length;
+}
+
+let calculateAvg = averageNumbers(numbersAvg);
```

Comment:
Since you are using the return statement and the function is not anonymous, it's not necessary to store the function in an additional variable